src/components/containers/popover/PopoverLink.js

```diff
@@ -65,7 +67,8 @@ class PopoverLink extends React.Component {
           }}
         >
           <PopoverButton
-            aria-haspopup="dialog"
+            aria-haspopup="true"
```

Comment:
What has been tried so far:

* `role="dialog"` on the PopoverContent
* `aria-labelledby` from the button to the content instead of `aria-describedby`
* `role="button"` on the button (for science)

Popovers are still unreadable to NVDA in firefox